packages/api-clients/open-api/notificationConfigApi.yml

```diff
@@ -95,6 +101,103 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/Problem"
+        "404":
+          description: Tenant notification config not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  /internal/tenantNotificationConfigs:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - process
+      summary: Create a tenant's notification configuration
+      operationId: createTenantNotificationConfig
+      requestBody:
+        description: A tenant notification config seed
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/TenantNotificationConfigSeed"
+        required: true
+      responses:
+        "200":
+          description: The tenant's newly created notification configuration
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/TenantNotificationConfig"
+        "400":
+          description: Invalid input
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "401":
+          description: Unauthorized
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "409":
+          description: Conflict
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  /internal/tenantNotificationConfigs/tenantId/{tenantId}:
```

Comment:
I used `/tenantId/{tenantId}` instead of just `/{tenantId}` since `tenantId` is not the main id of the `TenantNotificationConfig` entity but a sort of external id, though I think it's more convenient to use that instead of the tenant notification config id in the route (to match the update routes which get it from the token)

The path structure is the same as `/origin/{origin}/code/{code}` that we see in other services